README.md

```diff
@@ -1,7 +1,7 @@
 # Apache Spark
 
 Spark is a fast and general cluster computing system for Big Data. It provides
-high-level APIs in Scala, Java, and Python, and an optimized engine that
+high-level APIs in Scala, Java, Python, and R, and an optimized engine that
```

Comment:
R is listed in the first paragraph on [this page](http://spark.apache.org/docs/latest/index.html), so I figured it should be in the README.md as well.